File: scholar-web/src/main/webapp/app/model/administration/user/roles/PermissionSearch.js
Ext.define('scholar.model.administration.user.roles.PermissionSearch', {
	extend : 'Ext.data.Model',
	fields : [ {
		name : 'id',
		type : 'int'
	},{
		name : 'formName',
		type : 'string'
	}, {
		name : 'read',
		type : 'bool'
	}, {
		name : 'write',
		type : 'bool'
	}, {
		name : 'delete',
		type : 'bool'
	}, {
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ]
});

File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/PermissionSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class PermissionSearch extends AbstractEntity {
	String roleName;
	boolean read;
	boolean write;
	boolean delete;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}
